lua/acf/scanner/scanner_sh.lua

```diff
@@ -196,6 +198,62 @@ if SERVER then
     util.AddNetworkString("ACF_Scanning_PlayerListChanged")
     local scanningPlayers = {}
 
+    local scanner_blockedFromACFDamage = {}
+    local scanner_damageCooldown = 60 -- in seconds
+    local scanner_acfDamage_lastTick = 0
+    local scanner_acfDamage_notifiedThisTick = {}
+
+    function scanning.BlockACFDamage(playerScanning)
+        scanner_blockedFromACFDamage[playerScanning] = true
+    end
+    function scanning.StartUnblockACFDamage(playerScanning)
+        if scanner_blockedFromACFDamage[playerScanning] == true then
+            scanner_blockedFromACFDamage[playerScanning] = CurTime()
+        end
+    end
+    function scanning.UnblockACFDamage(playerScanning)
+        scanner_blockedFromACFDamage[playerScanning] = nil
+    end
+
+    hook.Add("ACF_PreDamageEntity", "ACF_Scanning_BlockDamageAfterScanner", function(ent, _, _)
+        if not IsValid(ent) then return end
+        local owner = ent:CPPIGetOwner()
+        if not IsValid(owner) then return end
+
+        local tickNow = engine.TickCount()
+        local doNotNotify = false
+
+        if tickNow ~= scanner_acfDamage_lastTick then
+            scanner_acfDamage_lastTick = tickNow
+            table.Empty(scanner_acfDamage_notifiedThisTick)
+        else
+            if scanner_acfDamage_notifiedThisTick[owner] then
+                doNotNotify = true
+            end
+        end
+
+        local started = scanner_blockedFromACFDamage[owner]
+
+        if started then -- started is boolean true if active, number started (in curtime) if left scanning
+            local now = CurTime()
+
+            if started == true or (now - started < scanner_damageCooldown) then
+                if not doNotNotify then
+                    NetStart("DamageBlocked")
```

Comment:
Use [`ACF.SendNotify`](https://github.com/ACF-Team/ACF-3/blob/master/lua/acf/core/utilities/util_sv.lua#L23) with `Sucess` set to false, it should do the exact same thing.